package.json

```diff
@@ -2,8 +2,22 @@
     "name": "dmi-editor",
     "displayName": "DMI Editor",
     "description": "Custom Editor for Byond DMI files",
-    "version": "1.0.6",
+    "version": "1.0.7",
```

Comment:
Hold on bumping the version for now, i'll cut a release after with new workflows

Reply:
```suggestion
    "version": "1.0.6",
```
Sorry